src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import filterReducer from '../features/filter';
+import todosReducer from '../features/todos';
+import currentTodoReducer from '../features/currentTodo';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
```

Comment:
The function `combineSlices` is not a valid function from Redux Toolkit. It should be `combineReducers`. Please replace `combineSlices` with `combineReducers` to correctly combine your reducers.